### DISCUSSION TITLE
Gerador de itens

### DISCUSSION THREAD

**Discussion by YuriAlessandro** [YuriAlessandro/gerador-ficha-tormenta20#57] [Ideias]
Acho que a gente poderia criar um gerador de itens também, podendo gerar armas com modificações, escudos, armaduras, poções, etc, usando as próprias combinações do livro

**Comment by RuanOniiKun** [comment]
Não tenho certeza se estou dando essa sugestão no lugar certo por que não conheço nem nunca usei essa plataforma, mas percebi que o druida está vindo como devoto de Allihanna, Oceano e Aharadak, rolei algumas vezes pra testar e sempre vem as três opções, acho que ficou trocado na programação o Aharadak e o Megalokk. Espero ter ajudado de alguma forma. Meus parabéns pelo site ficou muito foda usarei em breve!
